public/js/index.js

```diff
@@ -758,6 +799,64 @@ Index.migrateProgress = function () {
     }
 };
 
+/**
+ * Restore and update column width, data store in localstorge.
+ */
+Index.resizableColumns = function () {
+    let currentHeader;
+    let startX;
+    let startWidth;
+
+    const headers = document.querySelectorAll("#header-row th");
+    headers.forEach((header, index) => {
+        // restore Column Width
+        const savedWidth = localStorage.getItem(`resizeColumn${index}`);
+        if (savedWidth) {
+            header.style.width = savedWidth;
+        }
+        // init
+        header.addEventListener('mousedown', function (event) {
+            if (event.offsetX > header.offsetWidth - 10) {
+                currentHeader = header;
+                startX = event.clientX;
+                startWidth = header.offsetWidth;
+
+                document.addEventListener('mousemove', resizeColumn);
+                document.addEventListener('mouseup', stopResize);
+
+                document.body.style.cursor = 'col-resize';
+            }
+        });
+        header.addEventListener('mousemove', function (event) {
+            if (event.offsetX > header.offsetWidth - 10) {
+                header.style.cursor = 'col-resize';
+            } else {
+                header.style.cursor = 'default';
+            }
+        });
+    });
+
+    function resizeColumn(event) {
+        if (currentHeader) {
+            const newWidth = startWidth + (event.clientX - startX);
```

Comment:
This calculation is very inaccurate to me some times - I have to drag the mouse pretty far before the column width starts reacting (as seen in the screen recording in the other comment)  

Some other times, it will violently snap the column width to a very different value as soon as I start sliding.